src/zkevm_specs/evm/execution/error_stack.py

```diff
@@ -0,0 +1,55 @@
+from ...util import FQ
+from ..instruction import Instruction, Transition, FixedTableTag
+from ..table import CallContextFieldTag
+from ..execution_state import ExecutionState
+from ..opcode import Opcode
+from ...util import N_BYTES_STACK
+
+
+def stack_error(instruction: Instruction):
+    # retrieve op code associated to stack error
+    opcode = instruction.opcode_lookup(True)
+    # TODO: lookup min or max stack pointer
```

Comment:
Is this a leftover comment?

Reply:
yes, will remove it.

Reply:
remove TODO in [92d101b](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/301/commits/92d101b7ba811f7e71a441ad9272c7cd967ccf6f) thanks !